sympy/stats/tests/test_continuous_rv.py

```diff
@@ -733,6 +738,15 @@ def test_laplace():
     assert cdf(X)(x) == Piecewise((exp((-mu + x)/b)/2, mu > x),
                             (-exp((mu - x)/b)/2 + 1, True))
 
+def test_levy():
+    mu = Symbol("mu")
+    c = Symbol("c", positive=True)
+
+    X = Levy('x', mu, c)
+    assert X.pspace.domain.set == Interval(mu, oo)
+    assert density(X)(x) == sqrt(c/(2*pi))*exp(-c/(2*(x - mu)))/((x - mu)**(S.One + S.Half))
+    assert cdf(X)(x) == erfc(sqrt(c/(2*(x - mu))))
```

Comment:
Cover corner cases well, like, what if a complex `mu` is passed, a negative `c` is passed.
Is `mgf` raising the `NotImplementedError`? Add a test for that too.
Use, `raises` for checking if the proper exceptions are raised.

Reply:
mgf is not defined for this distribution. So can you please suggest how to add a test for that.

Reply:
> Cover corner cases well, like, what if a complex `mu` is passed, a negative `c` is passed.
> Is `mgf` raising the `NotImplementedError`? Add a test for that too.
> Use, `raises` for checking if the proper exceptions are raised.

Okay, I understood how to add that.

Reply:
For other corner cases I got to add the test cases. But for mfg
SingleContinuousPSpace calls a method from SingleContinuousDistribution which excepts NotImplementedError and return an lamda function of NoneType. So how should I add test for mgf because for some other distibutions whose mgf is not defined, test cases for them are not added in test_continuous_rv.py
```
def moment_generating_function(self, t, **kwargs):
        """ Moment generating function """
        if len(kwargs) == 0:

            try:
                mgf = self._moment_generating_function(t)
                if mgf is not None:
                    return mgf
            except NotImplementedError:
                return None
```

Reply:
I think that the message should reach to the end user that `mgf` of a distribution isn't defined. So probably, for now you can leave the test for mgf but we can leave the above issue for another discussion.
Please ensure that all the lines of a diff are covered.
What is the output for mgf of Levy distribution though?

Reply:
```
>>>mu = Symbol("mu", real=True)
>>>c = Symbol("c", positive=True)
>>>L = Levy("x", mu, c)
>>>moment_generating_function(L)
Lambda(_t, None)
```